remaining packer content (#13285)
hashicorp/packer: website/content/docs/commands/plugins/required.mdx

---
description: |

>   The `packer plugins required` command lists all plugins required in a Packer configuration as well as the installed binaries that match the constraint.

Reviewer: ```suggestion
  The `packer plugins required` command lists all of the plugins required by a Packer configuration and any installed binaries that satisfy the requirements.
```
